index.html

```diff
@@ -1,16 +1,95 @@
+        </ul>
+    </nav>
+    <main>
+      <div class="clear_fix">
+      </div>
+      <section class="tab-content" id="main">
+        <div id="separate">
+          <header id="logo">
+          <img alt="MOHAI Logo" src="styles/pictures/MOHAI_LOGO.png">
+          </header>
+        </div>
+        <h1>Welcome!</h1>
+          <p>
+            At the Bezos Center for Innovation, explore how innovation shaped our region and changed the world. Through interactive exhibits, community and educational programs, and first-person insights from leading innovators, discover Seattle's role as a nexus of big ideas and new directions - and ignite the innovator within! Generously supported by Jeff and MacKenzie Bezos, this is the place to discover the past and future of creativity and invention in our region.
+          </p>
+          <p>
+            What does innovation look like? Who innovates, where does it happen, and how do great ideas evolve? To find out more, tackle a challenge in the Idea Lab, discover a Seattle-made invention in the Patent Tree, and check out cutting-edge concepts in What's Next. Through lectures, special programs and changing displays, Seattle innovators will share their latest projects and invite you to take a look at the future as it unfolds.
+          </p>
+          <p>
+            Here in this app, you can explore some of the data from the exhibit! If you have visited the Bezos Center, you may have responded to questions throughout the exhibit. Now you can see what your friends and neighbors (near and far away!) have to say.
+          </p>
+      </section>
+      <section class="tab-content" id="about_visitors">
+        <h1>About our Visitors</h1>
+        <h3> MOHAI's visitors are from all over the world and of all different ages! Check out the data below!</h3>
+        <div id="i_frame">
+        </div>
+        <h3>Here's the ages of our visitors:</h3>
+        <div id="age_groups">
+        </div>
+        <!-- changed id -->
+      </section>
+      <section class="tab-content" id="visual_page">
+        <h1>Learn what MOHAI visitors have said!</h1>
+        <h3>Select a question below and learn what other MOHAI visitors have said! You can even filter the data to learn more. Cool!</h3>
+        <div id="questions">
+          <select id="question_selection">
+            <option value="">Select a question to see how others have responded!</option>
+          </select>
+        </div>
+        <section class="tab-content" id="filter_section">
+          <ul>
+            <li>
+              <select id="filter_li">
+                <option value="">Filter by Demographic</option>
+              </select>
+            </li>
+          </ul>
+        </section>
+        <div id="visual_data">
+        </div>
+      </section>
+    </main>
+    <footer>
+      &copy; CarRenDanJur 2016
+    </footer>
+    <script src="https://cdn.rawgit.com/visionmedia/page.js/master/page.js"></script>
```

Comment:
When moving to production, you should remove the protocols from your src/href type attributes. 
```html
<script src="//cdn.rawgit.com/visionmedia/page.js/master/page.js"></script>
```
This allows the machine to determine the default protocol, as well as adjust accordingly if that's not the correct protocol. 

Keep in mind that this requires using an active server environment when testing. If you open the file in the browser, using something like open-in-browser, the default protocol is `file://` which will not complete your CDN request.